ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPoolLondonTest.java

```diff
@@ -252,6 +254,25 @@ public void shouldAcceptLocal1559TxsWhenMaxFeePerGasIsAtLeastEqualToMinMinGasPri
         .isEqualTo(1);
   }
 
+  @Test
+  public void addRemoteTransactionsShouldNotTriggerIllegalStateException() {
```

Comment:
What is the condition that triggers the illegal state?

Reply:
Yes, definitely, what would be your suggestion ?

Reply:
Okay for addRemoteTransactionsShouldAllowDuplicates